esmvalcore/_recipe.py

```diff
@@ -401,11 +401,20 @@ def _get_correct_fx_file(variable, fx_varname, config_user):
         elif fx_varname == 'sftgif':
             fx_var = _add_fxvar_keys({'short_name': fx_varname, 'mip': 'fx'},
                                      var)
+        elif fx_varname == 'volcello':
+            fx_var = _add_fxvar_keys({'short_name': fx_varname, 'mip': 'Ofx'},
+                                     var)
     else:
         raise RecipeError(
             f"Project {var['project']} not supported with fx variables")
 
     fx_files = _get_input_files(fx_var, config_user)
+
+    # If can't find Ofx files, then try with original variable mip
+    if not fx_files and fx_var['mip'] == 'Ofx':
+        fx_var['mip'] = var['mip']
+        fx_files = _get_input_files(fx_var, config_user)
+
```

Comment:
maybe add a message here `logger.info("Tried looking for both Ofx and %s", var['mip'])`

Reply:
Done.